### ISSUE TITLE
Go to specific image or video when finishing playing a video

### ISSUE THREAD

**Issue by capturahd** [ChurchApps/FreeShow#528]
Hello, greetings, I have a question about an action that I don't see how to do in freeshow. A function that optimizes the work in some live production apps is that when finishing a video playback, you can assign an action to go to another source. image or other video.

Example: the video or image is not the following, it is a loop entered as the first medium and video 6 at the end I want it to go to video 1, which is the event loop.

**Comment by capturahd** [comment]
hello. I mean placing 10 videos and an infinite video loop, and at the end of each video, which is an interview, the video that is projected is video 11, which is the infinite loop, until they ask for the next video to play and at the end return to the video. 1 which is the loop.

**Comment by vassbo** [comment]
Okay, I get you.

You have e.g. 10 videos that sould play in sequence, and 1 looping video that should play as a loop in between, until e.g. space bar is then pressed, then continue to the next video.

I think the best would be to drag in the 10 videos, set the "Next after media" on all of them, then just one by one drag in the same loop video in between. Does that work for you?

**Comment by capturahd** [comment]
hi.

What you say is how I achieved it by needing this function (image 1), after my event, thinking about other programs like PVP, I tried to do the same without duplicating the loop so much (image 2), since we are talking about almost 20 videos and repeating This loop 20 times does not seem the most efficient in the long term.

Thanks for your answers, I understand that the option in image 1 is the only one at the moment to achieve this type of actions. Greetings from Panama

![1](https://github.com/ChurchApps/FreeShow/assets/89743434/891fcef9-9a1d-4651-9112-ab8236e404cc)

![2](https://github.com/ChurchApps/FreeShow/assets/89743434/3ea7c4b9-18af-4035-a13b-06e7895d91bb)

**Comment by capturahd** [comment]
hi.

What you say is how I achieved it by needing this function (image 1), after my event, thinking about other programs like PVP, I tried to do the same without duplicating the loop so much (image 2), since we are talking about almost 20 videos and repeating This loop 20 times does not seem the most efficient in the long term.

Thanks for your answers, I understand that the option in image 1 is the only one at the moment to achieve this type of actions. Greetings from Panama

![2](https://github.com/ChurchApps/FreeShow/assets/89743434/f11499dd-8cb1-43f8-8e33-3e60efda1b54)

![1](https://github.com/ChurchApps/FreeShow/assets/89743434/b0b2ffb4-732a-47de-8e63-db69816eadb2)
